.travis.yml

```diff
@@ -1,4 +1,4 @@
-osx_image: xcode7.1
+osx_image: xcode7.3
 language: objective-c
 
```

Comment:
That will fix the Error 65 stuff, but I noticed an actual _test_ failing intermittently (had to do with dates). I'll take a look.